.travis.yml

```diff
@@ -1,158 +1,278 @@
 language: php
 
+git:
+  strategy: clone
+  depth: 1
+  quiet: true
+
+dist: bionic
+
 php:
-  - 5.6
+  - 7.1
+  - 7.2
+  - 7.3
```

Comment:
Only min/max is needed, the other versions are already tested in matrix below quite extensively, from what we can see.

Reply:
We have edgecases everywhere in this project that cannot be found with minmax.

The matrix expansion of travis doesn't work well for this project anyway, I'd like to remove it so we can correctly test mysql versions.